flepimop/gempyor_pkg/src/gempyor/testing.py

```diff
@@ -30,3 +41,173 @@ def change_directory_to_temp_directory() -> Generator[None, None, None]:
     yield
     os.chdir(current_dir)
     temp_dir.cleanup()
+
+
+def create_confuse_configview_from_dict(
+    data: dict[str, Any], name: None | str = None
+) -> confuse.ConfigView:
+    """
+    Create a ConfigView from a dictionary for unit testing confuse parameters.
+
+    Args:
+        data: The data to populate the confuse ConfigView with.
+        name: The name of the Subview being created or if is `None` a RootView is
+            created instead.
+
+    Returns:
+        Either a confuse Subview or RootView depending on the value of `name`.
+
+    Examples:
+        >>> data = {
```

Comment:
Perhaps this is completely unnecessary but it could be good to insert a short sentence describing exactly what you're doing in the example call to the function?

Reply:
That's a good point, I've added a description with the yaml being represented.